evm_loader/cli/src/main.rs

```diff
@@ -872,32 +868,35 @@ fn main() {
                         let elf_params = get_neon_elf::read_elf_parameters_from_account(&config).unwrap();
                         Pubkey::from_str(elf_params.get("NEON_TOKEN_MINT").unwrap()).unwrap()
                     });
-                cancel_trx::command_cancel_trx(&config, &storage_account, &token_mint)
+                cancel_trx::execute(&config, &storage_account, &token_mint)
             }
             ("neon-elf-params", Some(arg_matches)) => {
                 let program_location = arg_matches.value_of("program_location");
 
-                get_neon_elf::command_neon_elf(&config, program_location)
+                get_neon_elf::execute(&config, program_location)
             }
             ("get-storage-at", Some(arg_matches)) => {
                 let contract_id = h160_of(arg_matches, "contract_id").unwrap();
                 let index = u256_of(arg_matches, "index").unwrap();
 
-                get_storage_at::command_get_storage_at(&config, &contract_id, &index)
+                get_storage_at::execute(&config, contract_id, &index)
             }
             ("update-valids-table", Some(arg_matches)) => {
                 let contract_id = h160_of(arg_matches, "contract_id").unwrap();
 
-                update_valids_table::command_update_valids_table(&config, &contract_id)
+                update_valids_table::execute(&config, contract_id)
             }
             _ => unreachable!(),
         };
+    
+    let exit_code: i32 =
         match result {
-            Ok(()) => exit(0),
-            Err(err) => {
-                error!("{}", err);
-                NeonCliError::UnknownError.report_and_exit();
-                // exit(NeonCliError::UnknownError as i32);
+            Ok(_)  => 0,
+            Err(e) => {
+                error!("{}", e);
+                e.error_code() as i32
```

Comment:
Please add `error_code` (as i32 or/and rust enum id) into logs
For example
".*cli exit error (i32 code) : <error_tesxt>"
Now:
```
2022-01-25 09:44:05402     ERROR     NA:Undefined         Emulator:neon_cli                       {'req_id': '17676ca'} Code account required at address 0x262503bef30787a513f9972155492ad495c39993
```

Reply:
Fixed

Reply:
Thanks. It's better:
```
2022-01-25 12:59:35131     ERROR     NA:Undefined         Emulator:neon_cli                       {'req_id': '98b6f1f'} NeonCli Error (206): Account not found at address 0xe1dab7a6176f8768f53a425f296173605ed50832.
...
2022-01-25 12:59:35168     ERROR     NA:Undefined         Emulator:neon_cli                       {'req_id': 'b2192bc'} NeonCli Error (208): Code account required at address 0x7f86d270107ca12d81133a1fcbb9aea30921a6c9.
```